Create `Gameplay/Multiplayer` to separate mechanics from the UI description (#15113)
ppy/osu-wiki: wiki/Client/Interface/Multiplayer/en.md

Reviewer: What happens if there's no slot available in the match room?

Author: nothing much — you are allowed into the room but no further, will add this later

Author: rechecked, and apparently this is not true -- it will say that the match is full

Reviewer 2: ```suggestion
| **Green (ready)** | The player is ready. Game modifiers cannot be changed in this state. The host will be able to start the game after readying, and all ready players will begin the match. |
```

Reviewer 2: ```suggestion
| `Last Played` | When the player last played the beatmap |
```

# Multiplayer

::: alert-note
**Note:** For how multiplayer works, see [Multiplayer](/wiki/Gameplay/Multiplayer)
:::

::: alert-note

> **See also:** [Quick play](/wiki/Gameplay/Quick_play) and [Daily challenge](/wiki/Gameplay/Daily_challenge)

Reviewer: `Quick play` does not exist anymore in the form the linked article describes it. Personally, I would remove it, but maybe okay to leave it in as an interim solution.

Author: going with the second option, yeah

:::

> This article covers the parts of the game client used for [multiplayer](/wiki/Gameplay/Multiplayer): the lobby, match setup, and the gameplay methanics.

Reviewer: ```suggestion
This article covers the parts of the osu!(stable) game client used for [multiplayer](/wiki/Gameplay/Multiplayer): the lobby, match setup, and the gameplay mechanics.
```

| `Start Game!` | Starts the match. This button only appears for the match host when all players are ready. |
| `Force Start Game! ({ready}/{total})` | Starts the match even if not all players are ready. This button only appears for the match host when some players are ready. The amount of ready players compared to the total amount of players is shown in parentheses. |

### Chat console

The bottom section of the screen is the [chat console](/wiki/Client/Interface/Chat_console). Every match has its own channel in a tab called `#multiplayer`, with [BanchoBot](/wiki/BanchoBot) providing a link to the [match history](/wiki/Gameplay/Multiplayer#match-history) on the first line.

## While playing

The gameplay screen itself is the same as in solo play, with the multiplayer leaderboard and a few team mode specific parts on top of it.

> While the match is starting, moving the cursor to the bottom of the screen will open the visual settings panel. See the [Visual settings](/wiki/Client/Interface/Visual_settings) page for more information.

Reviewer: ```suggestion
While the match is starting, moving the cursor to the bottom of the screen will open the [visual settings](/wiki/Client/Interface/Visual_settings) panel, allowing for visual customisations.
```

Maybe? 🤔

> The gameplay screen itself is the same as in solo play, with the multiplayer leaderboard and a few team mode specific parts on top of it.

Reviewer 2: ```suggestion
The gameplay screen itself is the same as in solo play, with the player's current placement in the bottom right, the multiplayer leaderboard and a few team mode specific parts on top of it.
```
or something like that
